concurrent/actor/Actor.java

```diff
@@ -44,14 +43,9 @@ protected State(Actor<STATE> self) {
         protected <CHILD_STATE extends State<CHILD_STATE>> Actor<CHILD_STATE> child(Function<Actor<CHILD_STATE>, CHILD_STATE> stateConstructor) {
             return child(self.eventLoop, stateConstructor);
         }
-
-        protected Actor<STATE> self() {
-            assert this.self != null : ERROR_SELF_ACTOR_IS_NULL;
-            return this.self;
-        }
     }
 
-    private STATE state;
+    public STATE state;
```

Comment:
Why is this made `public`?

Reply:
Because in the tests it would be easier to investigate the actor state (raft) directly instead of one layer of indirection.

Reply:
ok, let's keep it for now until we can refactor the test.